.docker/images/nginx/tests/src/RobotsTest.php

```diff
@@ -1,47 +0,0 @@
-<?php
```

Comment:
this file is renamed to `RobotsTxtTest.php` for consistency with `govcms8lagoon`.